### DISCUSSION TITLE
Multiple panels section doesn't seem to work anymore.

### DISCUSSION THREAD

**Discussion by holden** [swellstores/community#387] [Help] [answered]
Hi,

We noticed recently that it seems no longer possible to add new content blocks with `multiple panels` in the Origin theme. It still works when editing pre-existing ones. But new ones just appear blank and don't give you the options to choose the type on the second step (see image below). 

We haven't ever really modified any of the existing content blocks, so I don't think it's anything we did, it just stopped working after some of the changes to the editor. There are no errors in the console or anything noticeable; the submenus to add those panels it should have just no longer work.

How could we fix this or get this working again?

<img width="1338" alt="CleanShot 2023-09-26 at 13 59 13@2x" src="https://github.com/swellstores/community/assets/48241/9c612c8b-637b-4935-83fe-4b536ba7ea17">

https://github.com/swellstores/community/assets/48241/f661e2fc-9d07-4bfb-ad4a-8a2d6136f789

**Comment by holden** [comment]
Any ideas here?

**Comment by sags95** [comment]
@holden I don't believe we've made any recent changes to this, but I'll pass this on to the team to confirm.

**Reply by holden** [reply]
Thanks! We've never made any changes related to this, so it's odd that it doesn't work but it would be nice if it did!

**Reply by logeshswell** [reply, marked as answer]
I can see a bug already filed for it, I will keep you posted if I get some updates.

**Reply by holden** [reply]
Awesome thanks!

**Reply by LuigiNicaPRO** [reply]
I have exactly the same problem. Any new created panel can't have any content.
I don't know what's happening, as multipanel doesn't work, full width image shows only 30% ... it's unusable. 
I understand that they won't support it but why nothing works?
